docs/index.md

```
@@ -91,14 +93,43 @@ gunicorn -c gunicorn.conf.py your_app:app
+# Quick test with Docker Compose
+git clone https://github.com/Agent-Hellboy/gunicorn-prometheus-exporter.git
+cd gunicorn-prometheus-exporter
+docker-compose up --build
+
+# Or pull pre-built images
+docker pull princekrroshan01/gunicorn-prometheus-exporter:latest
+docker pull princekrroshan01/gunicorn-app:latest
+```
+
+**Available Services:**
+- Application: http://localhost:8000
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)